src/store/actions.ts

```diff
@@ -235,11 +235,11 @@ const commitExistingData = ({ commit, getters }) => {
     let { xproJurisdiction } = getters.getNr
     let location: Location
     for (let key of ['value', 'text']) {
-      if (CanJurisdictions.some(jurisdiction => jurisdiction[key].toUpperCase() === xproJurisdiction)) {
+      if (CanJurisdictions.some(j => j[key].toUpperCase() === xproJurisdiction.toUpperCase())) {
```

Comment:
When the NR is initially created, `xproJuridiction` is in mixed case (as in the list data).

However, after the NR has been edited (patched), `xproJuridiction` is in upper case.

This fix handles both cases.

(I changed the lambda to `j` to shorten the line / cleaner code.)